contribute.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "Mozilla Careers",
+    "description": "Lumbergh is the Mozilla careers website. It is a playdoh-based site that displays open jobs.",
+    "repository": {
+        "url": "https://github.com/mozilla/lumbergh",
+        "license": "BSD 3-Clause"
```

Comment:
I'm curious about the BSD 3-Clause license, I would have expected our code to be under the Mozilla Public License?

I suspect since you are doing a bunch of these this is the correct license but for my own education I was hoping you could explain.

Other than that all the links check out.